src/sentry/templatetags/sentry_api.py

```diff
@@ -21,14 +20,12 @@ def serialize(context, obj):
     else:
         user = None
 
-    return convert_to_json(serialize_func(obj, user), escape=True)
+    return convert_to_json(serialize_func(obj, user))
 
 
 @register.simple_tag
-def convert_to_json(obj, escape=False):
-    if escape:
-        return json.dumps_htmlsafe(obj)
-    return mark_safe(json.dumps(obj))
+def convert_to_json(obj):
```

Comment:
We should should stop using this method entirely. It's not used as a template tag from what i can tell. The `to_json` filter works now.